pkg/storage/export/export/export.go

```diff
@@ -950,6 +954,21 @@ func (ctrl *VMExportController) createExporterPodManifest(vmExport *exportv1.Vir
 		MountPath: "/token",
 	})
 
+	podManifest.Spec.Containers[0].ReadinessProbe = &corev1.Probe{
+		ProbeHandler: corev1.ProbeHandler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Scheme: corev1.URISchemeHTTPS,
+				Path:   ReadinessPath,
+				Port: intstr.IntOrString{
+					Type:   intstr.Int,
+					IntVal: 8443,
+				},
+			},
+		},
+		InitialDelaySeconds: 15,
+		PeriodSeconds:       10,
```

Comment:
these seem a little too long to me

Reply:
lowered them both to 5s

Reply:
The longer intervals are fine for long running system components that virt-operator installs but could be annoying for things that we want to quickly come and go like export pods